src/components/Redemptions/RedemptionsMenu.tsx

```diff
@@ -198,7 +198,7 @@ const SubscribedMenuItemContent = withSubscription({
     const { currentAccountAddress, proposal } = props;
     const arc = getArc();
     const dao = arc.dao(proposal.coreState.dao.id);
-    const daoEthBalance = concat(of(new BN("0")), await ethBalance(proposal.coreState.dao.id)).pipe(ethErrorHandler());
+    const daoEthBalance = concat(of(new BN("0")), await ethBalanceDao(proposal.coreState.dao.id)).pipe(ethErrorHandler());
```

Comment:
why the redemption page need to pole for eth balance ? should it just read that to see the dao balance and calculate if the redemption is possible.?

Reply:
the question is out of scope of this PR, but the answer is we poll in order to keep the value up-to-date